### ISSUE TITLE
How are 2D keypoints obtained

### ISSUE THREAD

**Issue by sunjc0306** [boycehbz/MvSMPLfitting#5]
Could you tell me some details about how to obtain 2D key points? 
is it obtained by oppose ?

**Comment by animesh-77** [comment]
> Could you tell me some details about how to obtain 2D key points? is it obtained by oppose ?

Yes I was able to get them by running OpenPose on all images. However, I had to change the ordering of keypoints stored in .json file
